lib/coins/btc.js

```diff
@@ -53,3 +54,13 @@ function validate (network, address) {
   if (bech32Validator(network, address, bech32Opts)) return true
   return false
 }
+
+function createWallet () {
+    const keyPair = bitcoin.ECPair.makeRandom()
+    const segwitAddr = bitcoin.payments.p2wpkh({ pubkey: keyPair.publicKey })
+
+    return {
+        publicAddr: segwitAddr.address,
+        privateAddr: keyPair.toWIF()
```

Comment:
👍

Reply:
1. This is actually a privateKey not an address.
2. I think we should standardize on full names like publicAddress, otherwise we can easily forget the name of the variable which leads to bugs.